activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+*   Added PostgreSQL migration commands for enum rename, add value, and rename value.
+
+    `rename_enum` and `rename_enum_value` are reversible though due to Postgres
+    limitation `add_enum_value` is not reversible since you cannot delete enum
+    values at this time and instead would have to drop and recreate the enum
+    entirely.
```

Comment:
```suggestion
    `rename_enum` and `rename_enum_value` are reversible. Due to Postgres
    limitation, `add_enum_value` is not reversible since you cannot delete enum
    values. As an alternative you would should drop and recreate the enum entirely.
```

Reply:
I've made this change, no more run on sentence, thank you!